client/signup/steps/domains/style.scss

```diff
@@ -54,6 +54,10 @@ body.is-section-signup.is-white-signup {
 	.domains__step-content {
 		display: flex;
 
+		@include break-small {
+			justify-content: center;
+		}
```

Comment:
Since the login & plans (sub)titles are centered on medium-sized viewports onwards (see Figma gnYBUOBthTMckJRJr3pZmk-fi-2624%3A237, gnYBUOBthTMckJRJr3pZmk-fi-2572%3A0 & gnYBUOBthTMckJRJr3pZmk-fi-2572%3A124), I applied it to the domain picker as well.